Frontend/src/AddUserForm.svelte

```diff
@@ -0,0 +1,88 @@
+<script>
+    let title;
+    let firstName;
+    let lastName;
+    let email
+
+    let location = [];
+    let group = [];
+
+    const handleSubmit = () => {
+        console.log("Submitted")
+    }
+
+</script>
+
+<main>
```

Comment:
<main> tags are intended to denote the main content of the entire page, and shouldn't be used for every component. 

_Also just so you know, unlike React, you don't need to have everything inside one element if you don't want to. I definitely do prefer it personally though_

Reply:
Fair, I think I just got into the motions of using it on all the components as I was working. Now would I want to use or avoid <main> in the modal? When the modal is activate it becomes the "main content" for the page.

Reply:
Your use of \<dialog\> is sufficient for your modals. I wouldn't consider them main content

_FYI, you can escape angle brackets like this in the future \\\<tagname\\\>_

Reply:
Okay, now to be specific on this project or other single page applications. The app file has a <main> tag which instantiates the other pages. Said pages also have a <main> tag. Is it better to have the <main> in the "app div" (App.svelte) remain as is, or be removed? Instead, would it be better to remove <main> from the pages? The <main> of the App.svelte to me feels like it is just more of the body. The <main> of a page feels like the main focus of the page. Wouldn't it be confusing, especially in an accessibility sense?

Reply:
Truly it does not matter which way you go about it - **from an accessibility standpoint**. Do whatever makes most sense to you, but I would personally find having the \<main\> tag wrapped around the page component to be easier to follow and easier to develop.

I think this is a really good question, so I'll make a small example here to kinda make sense of my thoughts on it. Imagine you have ParentFunction(), and ChildFunction1(), ChildFunction2(), ChildFunction3(), ... ChildFunctionN(). Would it make more sense to write them like this:

```
ParentFunction(){
   for(let i =0; i < ChildFunctions.length; i++){
      logger.log(\`executing order ${i}\`);
      ChildFunctions[i]();
   }
}

ChildFunction1(){
  //Child 1 stuff
}

ChildFunction2(){
  //Child 2 stuff
}

...

ChildFunctionN(){
  //Child N stuff
}
```

or like this:

```
ParentFunction(){
   for(let childFn of ChildFunctions){
       childFn();
   }
}

ChildFunction1(){
   logger.log("executing order 1");
   //Child 1 stuff
}

ChildFunction2(){
   logger.log("executing order 2");
   //Child 2 stuff
}

...

ChildFunctionN(){
   logger.log("executing order N");
   //Child N stuff
}
```

Reply:
I'd say the latter makes more sense, at least in terms of syntax. However, I see both as executing the child functions. The first has more variability in it's logger, while the second has more customizability. I still find the second to be the better version for it's organization.

Reply:
Very fair. In your case, I think flexibility (in terms of the \<main\> tag) isn't really a concern (yet), so I would lean towards the DRYer one.